### ISSUE TITLE
SL4010e dec/enc CP issue

### ISSUE THREAD

**Issue by dk5ras** [george-hopkins/codeplug#2]
Hi,

when I read my SL4010e EMEA, decode the codeplug and encode it again, I get an error 1214 that features are not purchased that I wanted to write. The radio has a few paid features (RX gain leveling, TTS. AES) legally activated. FW 2.80, CPS 15.5. I can send the codeplug privately if of any help, just do not want to share it in the public.

BTW, the same procedure with the DP4801e EMEA, same FW and CPS and paid features, works without any issues.

With best regards

Ralph.

**Comment by george-hopkins** [comment]
Hi Ralph

If you could send me the original and modified files to george-hopkins@null.net that would be great. As you can decode/encode files for DP4801e, it's probably a bug in codeplug rather than a problem with your setup.

**Comment by dk5ras** [comment]
Did you receive the files?

**Comment by george-hopkins** [comment]
Yes, thank you for sending them! It seems the pieces that cause the error seem to be a nested quite a bit. Just to verify that we observe the same behaviour: You are able to open the modified file successfully and the error is shown when you actually try to write the file, right?

**Comment by dk5ras** [comment]
Yes, this is the behavior, at some low percentage while writing the CPS throws an error. The radio is unimpressed from this, it does not get bricked.
